saviSimulation/config.cfg

```diff
@@ -14,10 +14,10 @@ RANDOM_SEED = 137
 REASONING_CYCLE_PERIOD = 5
 TREE_SIZE = 50
 HOUSE_SIZE = 15
-THREAT_SIZE = 10
+THREAT_SIZE = 15
 ANTENNA_SIZE = 26
 UGV_SIZE = 26
 UAV_SIZE = 26
-SENSORS_ERROR_PROB = 0.3
-SENSORS_ERROR_STD_DEV = 20
+SENSORS_ERROR_PROB = 0
+SENSORS_ERROR_STD_DEV = 0
```

Comment:
I suspect that we don't actually want these changes to the config file. Do we?

Reply:
I rather leave it this way till everything works to make sure the errors are not generating noise in the intended behavior.